### PR TITLE
Update v0.14.0 changelog

### PR DESCRIPTION
**Pull request by gkleiman**
cc/ @meichstedt

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -72,6 +76,10 @@ If an application requires IP-per-task, then it can not request ports to be allo
 Currently, this feature does not work in combination with Docker containers. We might still change some
 aspects of the API and we appreciate your feedback.
 
+#### Improved migrations
+It is now possible to resume interrupted ZooKeeper state migrations. That means that Marathon can now recover from
+transient errors during the migration process.
```
Comment (kolloch) [inline changelog.md]: Fact is it only applies to some specific migrations. I am not sure if it is worth it to clarify this.

Comment (kolloch) [inline changelog.md]: This sounds as if all migrations were not recoverable.

**Comment by kolloch** [comment]
Looks good to me!

I didn't check for completeness (should I?), though.

**Comment by kolloch** [comment]
Thx!

**Comment by meichstedt** [comment]
:+1:

**Pull request merged**